src/Client/Core/Entities/CleanEntityStorage.cs

```diff
@@ -6,29 +6,29 @@ namespace Microsoft.DurableTask.Client.Entities;
 /// <summary>
 /// Request struct for <see cref="DurableEntityClient.CleanEntityStorageAsync"/>.
 /// </summary>
-public readonly record struct CleanEntityStorageRequest
+public record CleanEntityStorageRequest
```

Comment:
I am not a big fan of changing to a class here, but not sure what a good alternative would be.

Reply:
Right, I agree. I couldn't figure out how to get the defaults specified without turning it into a class.

Reply:
I suppose I could define the struct without defaults and then define a static member `Default` which gets used as the default for the call. Let me try that.

Reply:
I have added a static `Default` factory method as a way to specify the defaults and still keep this a struct.